external/nghttp2/nghttp2_config.h

```diff
@@ -0,0 +1,399 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define if building universal (internal helper macro) */
+/* #undef AC_APPLE_UNIVERSAL_BUILD */
+
+/* Define to 1 to enable debug output. */
+/* #undef DEBUGBUILD */
+
+/* Define to 1 if you have the `accept4' function. */
+#define HAVE_ACCEPT4 1
+
+/* Define to 1 if you have the <arpa/inet.h> header file. */
+#define HAVE_ARPA_INET_H 1
+
+/* Define to 1 if you have the std::atomic_* overloads for std::shared_ptr. */
+/* #undef HAVE_ATOMIC_STD_SHARED_PTR */
+
+/* define if the Boost library is available */
+/* #undef HAVE_BOOST */
+
+/* define if the Boost::ASIO library is available */
+/* #undef HAVE_BOOST_ASIO */
+
+/* define if the Boost::System library is available */
+/* #undef HAVE_BOOST_SYSTEM */
+
+/* define if the Boost::Thread library is available */
+/* #undef HAVE_BOOST_THREAD */
+
+/* Define to 1 if your system has a working `chown' function. */
+#define HAVE_CHOWN 1
+
+/* define if the compiler supports basic C++11 syntax */
+#define HAVE_CXX11 1
+
+/* Define to 1 if you have the declaration of `initgroups', and to 0 if you
+   don't. */
+#define HAVE_DECL_INITGROUPS 1
+
+/* Define to 1 if you have the declaration of `strerror_r', and to 0 if you
+   don't. */
+#define HAVE_DECL_STRERROR_R 1
+
+/* Define to 1 if you have the <dlfcn.h> header file. */
+#define HAVE_DLFCN_H 1
+
+/* Define to 1 if you have the `dup2' function. */
+#ifndef __TIZENRT__
+#define HAVE_DUP2 1
```

Comment:
I think just commenting HAVE_DUP2 is enough without __TIZENRT__
because every platform will have their own.

Reply:
every platform will have their own, meaning other platforms wudn't use HAVE_DUP2 at all?

Reply:
It would be better to modify it as below
/* #define HAVE_DUP2 1 */
because config.h is an auto-generated file

Reply:
@pillip8282 Ok. done